### ISSUE TITLE
Stamina won't drain while pushing the jump for a short amount of time

### ISSUE THREAD

**Issue by Kaitaki** [PaperScorpio/sprint-o-meter#8]
Due to the config option "Stamina Deduction Delay" and how it works to slow down the "rate of deduction", it also causes checking movement (sprinting/jumping) to slow and ends up causing jumping to not deduct a point. 

This would generally not be an issue as the default value is a 3 tick delay and pressing the space bar within that time gap is rare. However, if someone were to turn up the delay, issues start occurring which is why the max limit for the delay will be set to 10 instead of 100. (Over 10 ticks the stamina system starts becoming an annoyance anyways, but if you want the stamina to deduct slower, consider turning up the max stamina).

tldr: won't fix, issue is minuscule and does not affect the quality of gameplay.
